### ISSUE TITLE
Too low fps on a high end laptop pls help!

### ISSUE THREAD

**Issue by kits2408** [ValveSoftware/Dota2-Gameplay#19048]
### Description

Since summer client update last year till today, my fps has been off, vsync is not working on window/borderless window. Gameplay is so stuttery (jittery) because of these low fps. Either playing on fullscreen it still stutters and have low fps but somehow vsync can be enabled. Lately I've installed msi afterburner and observed my parameters where it showed i only have MINIMUM FRAMES of 2 FPS & 1% low of 3 FPS. CPU usage is low, hovering only on 15-20% w/ temps of not over 90. All of my games are running smoothly except for dota. I watched benchmarking vids on YT where other shows higher min. frames & 1% low fps with lower specs than mine. 10 months trying and finding out solutions only to fail. Your suggestions/comments will be gladly appreciated! 

My specs are: 
i9-13900hx
RTX 4080 (laptop)
16gb 5600mhz - Waiting for my 32 gb ram to arrive this week

### Screenshots

![Screenshot 2024-06-02 150139](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144519770/6e9434fb-7793-40f8-a520-a1beba942714)

**Comment by TpsArchyy** [comment]
after 7.36 game started stuttering, that is what you are seeing i suppose.... lots of people are having the same issue..

**Comment by XkTheMaster** [comment]
Not only you have this problem , since 7.35 dota suffers from everything related to fps , low fps , big/small stutters , inconsistent framerate. Its been like this for quite some time , we need a spring clean update , otherwise I dont see a future for dota relating fps. About vsync yes, its broken since 2023 summer update, you cannot use it on window/borderless (only if you force it via AMD/Nvidia program)

**Comment by kits2408** [comment]
> Not only you have this problem , since 7.35 dota suffers from everything related to fps , low fps , big/small stutters , inconsistent framerate. Its been like this for quite some time , we need a spring clean update , otherwise I dont see a future for dota relating fps. About vsync yes, its broken since 2023 summer update, you cannot use it on window/borderless (only if you force it via AMD/Nvidia program)

Yeah we need a spring clean update so bad, Im just wondering why the hell i have these low minimum frames than other low specs hardware and i think this is causing my not so smooth gameplay experience. Compare to other games minimum frames are a bit lower than the avg frames i got. In Dota minimum frames of 2-3 fps in an avg of 200 fps? Valve need to fix this shit.

**Comment by IgorHoruzha** [comment]
@kits2408 Hello. Does Your issue look like this one #15078 ?

If You use Microsoft Defender Antivirus, could You try to turn off the real-time protection and check, please?
Could You try to turn off hyperthreading in BIOS and check, please?

Could You provide a video of the stutters, please?

It may be useful to get as much as possible information to produce the issue on purpose. It will give a roadmap to developers and essentially increase the chances of fixing the issue(at least fast). Do You have any sequence of actions that will lead to  stutters  arise with high chances?
